build.gradle.kts

```diff
@@ -31,16 +31,19 @@ subprojects {
         apply<DistributionPlugin>()
 
         val sep = File.separator
+        // The path where we want publishToMavenLocal to output the artifacts to
+        val buildPublishDir = "${project.layout.buildDirectory.get().asFile.path}${sep}sentry-local-publish$sep"
 
         configure<DistributionContainer> {
-            this.configureForMultiplatform(this@subprojects)
+            configureForMultiplatform(this@subprojects, buildPublishDir)
         }
 
         tasks.named("distZip").configure {
-            this.dependsOn("publishToMavenLocal")
-            this.doLast {
+            System.setProperty("maven.repo.local", buildPublishDir)
```

Comment:
so this will tell the task where to publish?

Reply:
yep, it then publishes into the specified dir